### DISCUSSION TITLE
Week 2 – Working on the basics

### DISCUSSION THREAD

**Discussion by Mohmad-Naser-alnakeshbandi** [The-Escape-Box/TheEscapeBox#20] [General]
In this week, we focused on finishing a playable prototype to demonstrate the core game mechanics.

# Weekly Progress (Sprint#1)
Our goals for this week are:
- Choose work distribution technology
- Choose version control for code
- Create a game environment
- Create a player
- add a gun and game shop
- add some enemies 

First enemy: 

![image](https://github.com/The-Escape-Box/TheEscapeBox/assets/76824323/8b8ee906-27f5-477c-955f-55d03476a53f)

Second enemy: 

![image](https://github.com/The-Escape-Box/TheEscapeBox/assets/76824323/47dcc0be-1a8c-4800-aede-3a9e37f58d47)

--- 
Time
Mohammad: 10h
Lukas: 10h

---

Please feel free to share your thoughts on the project in this and all subsequent posts. We will consistently update you on the steps we have taken and our plans for the future. Our aim is to fulfil both our own and your expectations.

**Comment by tonoizer** [comment]
Kudos on achieving your goals for Sprint #1! Building a playable prototype is a significant milestone. Excited to see the progress with the game environment, player dynamics, and the introduction of enemies. Keep up the fantastic work, looking forward to witnessing 'The Escape Box' come to life!

Team: AetherArena

**Comment by TimWortmann** [comment]
Wow, it's so exciting to see the progress you've made in just one week! The commitment and focus your team has put into creating a playable prototype are truly commendable. I love how transparent you are about your goals for the week. It gives us, as readers and potential players, a clear insight into the direction of the project. Choosing the right technology for work distribution and version control is crucial, and I'm glad to see you're tackling these aspects head-on. The visuals of the game environment and characters look fantastic already! The addition of the player, gun, game shop, and enemies really brings the game to life. Those enemy designs are both intriguing and a little bit intimidating, which I think adds to the excitement of what's to come.

Team 
Cooking Souls
